doc/rados/configuration/ceph-conf.rst

```diff
@@ -561,33 +561,32 @@ testing purposes, and are not recommended for use by operators.
 Runtime Changes
 ===============
 
-In most cases, Ceph allows you to make changes to the configuration of
-a daemon at runtime. This capability is quite useful for
-increasing/decreasing logging output, enabling/disabling debug
-settings, and even for runtime optimization.
+In most cases, Ceph permits changes to the configuration of a daemon at
+runtime. This can be used for increasing or decreasing the amount of logging
+output, for enabling or disabling debug settings, and for runtime optimization.
 
-Generally speaking, configuration options can be updated in the usual
-way via the ``ceph config set`` command.  For example, do enable the debug log level on a specific OSD:
+Configuration options can be updated via the ``ceph config set`` command.  For
+example, to enable the debug log level on a specific OSD, run a command of this form:
 
 .. prompt:: bash $
 
    ceph config set osd.123 debug_ms 20
 
-Note that if the same option is also customized in a local
-configuration file, the monitor setting will be ignored (it has a
-lower priority than the local config file).
+.. note:: If an option has been customized in a local configuration file, the
+   monitor setting will be ignored (it has a lower priority than the local
```

Comment:
Suggest s/monitor/central config/

Reply:
@anthonyeleven What do you think about all of the documentation being updated so that references to ceph.conf are replaced with references to the central configuration stored on the monitors? (Don't worry, I'm not suggesting that you do this. This would be a project that I would undertake over several months.)

Reply:
That occurred to me at one point but the scope of course is huge.  Central config came out in .... Mimic I think it was.  We do get folks running older releases that find more recent docs, so I try when feasible to mention first/last applicable release context.  That could get tedious, admittedly.  It's also the case that some deployments still use local conf files as well, where there is precedence to consider.

I've mused over the idea of factoring this all out into whatever sort of snippet or macro that RST might support, like we do now with the definitions of the config options themselves.